src/avram/migrator/column_default_helpers.cr

```diff
@@ -1,5 +1,5 @@
 module Avram::Migrator::ColumnDefaultHelpers
-  alias ColumnDefaultType = String | Time | Int32 | Int64 | Float32 | Float64 | Bool | Symbol
+  alias ColumnDefaultType = Object
```

Comment:
Hmm.. Not sure about this. It is technically cleaner, but one goal of Avram is to be VERY typesafe, and this would technically allow something like `Tuple` or `Point` to match because `Tuple.is_a?(Object)`.  @paulcsmith what are your thoughts on this?